gprofiler/merge.py

```diff
@@ -238,6 +239,7 @@ def concatenate_profiles(
     add_container_names: bool,
     metadata: Metadata,
     metrics: Metrics,
+    add_application_names: bool = True,
```

Comment:
Did you mean
```suggestion
    add_application_name: bool = True,
```
?

Reply:
Oh I now see that there is a `add_container_names` parameter. I don't understand this name...